Packs/Genians/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Genians",
+    "description": "Use the Genians integration to block IP using assign tag and unassign tag.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Genians",
+    "url": "https://www.genians.com/",
+    "email": "support@genians.com",
```

Comment:
@genians-rnd is this a valid email to get support from?

Reply:
Email is integration@genians.com.

I clicked wrong by my mistake.

Reply:
So let's change this accordingly

Reply:
I changed it.